tools/post_compute_crc.py

```diff
@@ -109,6 +112,23 @@ def main():
     with open(output_file_txt, 'wb') as file_handle:
         file_handle.write("0x%08x\r\n" % crc32)
 
+    # Write out data as a C array
+    data = new_hex_file.tobinarray(start=start, size=size)
+    data = list(bytearray(data))
+    output_data = bytearray('static const unsigned int image_start = 0x%08x;\n'
+                            'static const unsigned int image_size = 0x%08x;\n'
+                            'static const char image_data[0x%08x] = {\n    ' %
```

Comment:
The image data doesn't have any alignment requirements, since the flash manager code copies potentially unaligned data to an aligned buffer in RAM before flashing.